sql/internet_outages/global_outages_v1/query.sql

```diff
@@ -0,0 +1,525 @@
+-- Note: udf.udf_json_extract_int_map map doesn't work in this case as it expects an INT -> INT
+-- map, while we have a STRING->int map
+CREATE TEMP FUNCTION udf_json_extract_string_to_int_map(input STRING) AS (
+  ARRAY(
+    SELECT
+      STRUCT(
+        CAST(SPLIT(entry, ':')[OFFSET(0)] AS STRING) AS key,
+        CAST(SPLIT(entry, ':')[OFFSET(1)] AS INT64) AS value
+      )
+    FROM
+      UNNEST(SPLIT(REPLACE(TRIM(input, '{}'), '"', ''), ',')) AS entry
+    WHERE
+      LENGTH(entry) > 0
+  )
+);
+
+-- TODO: add docs
```

Comment:
@saptarshiguha could you kindly give me a couple of sentences to add as docs, here?

Reply:
this is a copy of https://github.com/mozilla/bigquery-etl/blob/master/udf/json_extract_int_map.sql#L8

Reply:
@saptarshiguha this is referring to the `sum_values` function below. Was this from some UDF as well?

Reply:
No sum values is of my own creation :)

Reply:
> 
> 
> this is a copy of https://github.com/mozilla/bigquery-etl/blob/master/udf/json_extract_int_map.sql#L8

Ok, because I was confused by your message quoted here. Can you kidnly provide a line of comment with respect to what this is doing?